AndroidBound/src/main/java/solutions/alterego/androidbound/ViewBinder.java

```diff
@@ -75,7 +73,7 @@
 
     private IImageLoader mImageLoader = IImageLoader.nullImageLoader;
 
-    public ViewBinder(Context ctx, IAndroidLogger logger) {
+    public ViewBinder(Context ctx, ILogger logger) {
```

Comment:
I would remove this dependency from the Constructor and add a setter as was done for IImageLoader. Having it in the constructor seems to me too strong as constraint. It is like saying that you can't use ViewBinder without a logger, which doesn't feel right